src/features/people/PersonPage/PersonDetails/styled.js

```diff
@@ -29,28 +30,31 @@ export const Wrapper = styled.section`
 
 export const Name = styled.header`
     grid-area: name;
+    margin: 8px 0 0;
     font-weight: 600;
     font-size: 36px;
     line-height: 120%;
-    margin: 8px 0 0;
 
-    @media (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
-        font-weight: 500;
-        font-size: 14px;
-        line-height: 130%;
```

Comment:
I dont get why those styles were removed. They were ok, also font-weight should be a bit less bold when we are using mobile device.

Reply:
maybe its my mistake i added again line height and font-weight thanks